server/api/src/rs_get_rescinfo_for_get.cpp

```diff
@@ -0,0 +1,94 @@
+#include <cstring> // for strdup.
+#include <nlohmann/json.hpp> // for strdup.
+
+//// ----------- TODO: make this file
+// #include "irods/rs_get_rescinfo_for_put.hpp"
+
+#include "irods/catalog_utilities.hpp"
+#include "irods/irods_logger.hpp"
+#include "irods/rodsErrorTable.h"
+
+#include "irods/getHostForPut.h" // for definition of macro: THIS_ADDRESS
+#include "irods/getHostForGet.h"
+#include "irods/rodsLog.h"
+#include "irods/rsGlobalExtern.hpp"
+#include "irods/rcGlobalExtern.h"
+#include "irods/getRemoteZoneResc.h"
+#include "irods/dataObjCreate.h"
+#include "irods/objMetaOpr.hpp"
+#include "irods/resource.hpp"
+#include "irods/collection.hpp"
+#include "irods/specColl.hpp"
+#include "irods/miscServerFunct.hpp"
+#include "irods/rsGetHostForGet.hpp"
+
+// =-=-=-=-=-=-=-
+#include "irods/irods_resource_backport.hpp"
+#include "irods/irods_resource_redirect.hpp"
+
+
+namespace
+{
+    using log_api = irods::experimental::log::api;
```

Comment:
This can be moved into the body of the function where it's used.

Reply:
Evidently this has disappeared since going to the newer GET_RESOURCE_INFO_FOR_OPERATION .